src/baguetteBox.js

```diff
@@ -66,6 +66,10 @@
     var imagesElements = [];
     // Event handlers
     var imagedEventHandlers = {};
+    // Contains the current image
+	var imageElement;
```

Comment:
Might as well follow the whitespace rules the rest of the code is using, no tabs, 4 spaces.